src/components/Card.jsx

```diff
@@ -5,18 +5,19 @@ import { CardTitle } from './styles/CardTitle.styled';
 import { CardTitleContainer } from './styles/CardTitleContainer.styled';
 import { CardDescription } from './styles/CardDescription.styled';
 import { CardDescriptionContainer } from './styles/CardDescriptionContainer.styled';
+
 import PropTypes from 'prop-types';
 
-const Card = ({ thumbnail, title, description, url }) => {
+const Card = ({ thumbnail, title, description, url, textColor }) => {
   return (
     <Link to={url}>
       <StyledCard>
         <CardThumbnail src={thumbnail} alt="video-thumbnail" />
         <CardTitleContainer>
-          <CardTitle>{title}</CardTitle>
+          <CardTitle textColor={textColor}>{title}</CardTitle>
```

Comment:
i'm trying to think if it is a good idea to get in this component directly the dark mode